### ISSUE TITLE
Release 1.8 still beta?

### ISSUE THREAD

**Issue by JonasVermoesenJdn** [collectiveaccess/providence#1413]
Hi we are looking to update our collective access website/api to the latest version. When we look at the given documentation we can see that the latest version should be 1.8. But this git repo says that the latest release is 1.7.17. We can see the 1.8 branch (https://github.com/collectiveaccess/providence/tree/1.8), is this a production ready version or is it still in beta?

**Comment by willisplummer** [comment]
I agree that it is very unclear -- all of the documentation references 1.8 and the last official news said that this version was probably going to be released in January or February. Any updates?

**Comment by collectiveaccess** [comment]
We are still finalizing PHP 8.2 support. It's close. Expect it before the end of April.

**Comment by JonasVermoesenJdn** [comment]
Hi. Any updates on the upcoming release?

**Comment by In2Creation** [comment]
Hi, we are also interested in version 1.8. Any issues with the release?

**Comment by JonasVermoesenJdn** [comment]
Hi, Any Updates?

**Comment by willisplummer** [comment]
Sorry to spam this thread, but a quick update would be really beneficial. Just trying to make some strategic decisions about an upcoming project and whether to use 1.7, push ahead with the latest RC of 1.8, or wait until 1.8 officially comes out

**Comment by JonasVermoesenJdn** [comment]
@willisplummer We had the same Issue and decided to keep 1.7 for now.

**Comment by andrea-favarin** [comment]
we are waiting for the official 1.8 release because some developments we have to do depend on bugs fixed on 1.8 and the development release seems to be moving every day

**Comment by collectiveaccess** [comment]
We prioritize the needs of the institutions we partner with, adding the features they require. This is not always in the order, or on the timescale, that we'd prefer. This is why the develop branch is constantly moving. 

We will do a release when a window occurs that allows us to spend the time to prepare it. We realize a release is a priority for you, but it's something that has to be fit within the larger context of the work we are required to do. 

If anyone from the user community wished to step up and assist with preparation of a release we'd welcome it.

**Comment by JonasVermoesenJdn** [comment]
Thank you for the clarity. We can work with the 1.7 release for now and upgrade in the future. We appreciate the hard work that is being done by this community. We only needed to have some clarity wish we now have.

**Comment by Monica-Wood** [comment]
> If anyone from the user community wished to step up and assist with preparation of a release we'd welcome it.

Seth, if you could document somewhere , or just dot point your steps to prepare a release I would be happy to assist.

**Comment by willisplummer** [comment]
I'd also be happy to help cut the release if we can get some more info about the steps required!

**Comment by poplarmedia** [comment]
I am also available to help with the 1.8 release too.
